library/mariadb/Chart.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: mariadb
+version: 0.0.1
+upstream_version: 0.1.1
+appVersion: "auto"
+description: The open source relational database
```

Comment:
Perhaps "An" instead of "The"?

"The" would generally mean it's the main contender.  For MariaDB, that's not the case. :wink:

Reply:
Afaik MariaDB is still the main contender.

Reply:
SQLite is **far** more widely deployed than both of them put together. :wink: